astropy/convolution/tests/test_convolve.py

```diff
@@ -889,15 +889,14 @@ def test_asymmetric_kernel(boundary):
 @pytest.mark.parametrize('ndims', (1, 2, 3))
 def test_convolution_consistency(ndims):
 
-    np.random.seed(0)
-    array = np.random.randn(*([3]*ndims))
-    np.random.seed(0)
-    kernel = np.random.rand(*([3]*ndims))
+    array = np.ones(shape=[3] * ndims)
+    kernel = np.zeros(shape=[3] * ndims)
+    kernel[..., 1] = 0.5
```

Comment:
Admittedly, this change didn't need to happen here. But it occurred to me that this really didn't need to be random values...